README.md

```diff
@@ -30,7 +30,7 @@
 
 # 📖 Awesome Books <a name="about-project"></a>
 
-> This project is about implementing webpack.
+> This project is about a list of tasks to do where the user will be able to add, remove and edit his tasks. All his data will be in local storage.
```

Comment:
Please, let's fix the broken link to your project license.  For a professional repository, let's avoid broken links in our readme 🤗👍

![image](https://github.com/EvansSnave/to-do-list/assets/78251927/e5b56326-83a7-409c-9af2-253537b005f5)